code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -243,47 +231,27 @@
 	var/list/minds = list()
 	///Tracked image
 	var/image/img
+	invisibility = INVISIBILITY_OBSERVER
 
 /obj/effect/reality_smash/Initialize()
 	. = ..()
-	img = image(icon, src, image_state, OBJ_LAYER)
+	//var/img = image(icon, src, "reality_smash", OBJ_LAYER)
 	generate_name()
+	// This was on TG but I'm not sure what the bee equivalent is --- add_alt_appearance(/datum/atom_hud/alternate_appearance/basic/heretics,"influence",img)
```

Comment:
this is the shit that makes it invisible to non-heretics I'm pretty sure

Reply:
Ah; it should work now properly.